website/templates/base.mako

```diff
@@ -173,6 +173,15 @@
             </script>
         % endif
 
+        %if keen_project_id:
+            <script>
+                window.contextVars = $.extend(true, {}, window.contextVars, {
+                    keenProjectId: '${keen_project_id}',
```

Comment:
Glancing at how other contextVars are declared, it looks like the proper way to JSON-encode variables in our mako templates is:

```mako
keenProjectId: ${ keen_project_id | sjson, n }
```